README.md

```diff
@@ -11,6 +11,7 @@ Homebrew tap for AMIA Open Source formulae:
 - **gtkdialog** — A small utility for fast and easy GUI building. Note that gtkdialog is used by vrecord.
 - **LTOpers** — Scripts for doing stuff with LTFS.
 - **mkvparse** — A simple Matroska (MKV) parser.
+- **mpv** - A cross platform media player (forked from https://github.com/mpv-player/mpv)
```

Comment:
nit: the others are m–dashes

Reply:
updated in https://github.com/amiaopensource/homebrew-amiaos/pull/204/commits/8b846b60d07e19bd59461dcf5edb4b6a0613eec5